app/models/site.rb

```diff
@@ -0,0 +1,12 @@
+class Site < Base
+  belongs_to :recruitment_cycle, through: :provider, param: :recruitment_cycle_year
+  belongs_to :provider, param: :provider_code
+  has_one :site_status
+
+  properties :code, :location_name, :address1, :address2, :address3
+  properties :address4, :postcode, :latitude, :longitude
+
+  def full_address
+    [address1, address2, address3, address4, postcode].select(&:present?).join(", ").html_safe
+  end
```

Comment:
Feels like this should be in the decorator, since it deals with outputting HTML.

Reply:
Same as frontend, leaving as is